packages/boilerplate-generator-tests/test-lib.js

```diff
@@ -50,5 +52,7 @@ export function generateHTMLForArch(arch) {
     },
   });
 
-  return boilerplate.toHTML();
+
+  return await toString(boilerplate.toHTML());
```

Comment:
No need to explicitly `await` when returning a value that could be a `Promise` from an `async` function.